docs/api/rest-endpoints.md

```diff
@@ -100,3 +114,384 @@ Example
   ]
 }
 ```
+*******
+
+## Entrance
+
+### Log in
+
+`POST /api/v1/kolide/login`
+
+#### Parameters
+
+| Name                    | Type    | In    | Description                                       |
+|-------------------------|---------|-------|---------------------------------------------------|
+| username                | string  | body  | **Required**. The user's email.              |
+| password                | string  | body  | **Required**. The user's plain text password.|
+
+#### Example
+
+`POST /api/v1/kolide/login`
+
+##### Request body
+
+```
+{
+  username: "janedoe@example.com"
+  passsword: "VArCjNW7CfsxGp67"
+}
+```
+
+##### Default response
+
+`Status: 200`
+
+```
+{
+  "user": {
+    "created_at": "2020-11-13T22:57:12Z",
+    "updated_at": "2020-11-13T22:57:12Z",
+    "id": 1,
+    "username": "jane",
+    "name": "",
+    "email": "janedoe@example.com",
+    "admin": true,
+    "enabled": true,
+    "force_password_reset": false,
+    "gravatar_url": "",
+    "sso_enabled": false
+  },
+  "token": "{your token}"
+}
+
+```
```

Comment:
Can we add a note in here saying something like the following?

> Use the token returned from this endpoint to authenticate further API requests.

We probably need to add an intro to this document that explains the format that's expected for the `Authorization` header.

Reply:
Drafted an introduction to the document that walks the user through authentication and making an authenticated request.

I included your exact note with a description of the `/login` endpoint. I also included a modified version of the note in the document's introduction.